docs/reference/ilm/index.asciidoc

```diff
@@ -5,17 +5,19 @@
 
 [partintro]
 --
-You can configure {ilm} ({ilm-init}) policies to automatically manage indices according to
-your performance, resiliency, and retention requirements. 
+You can configure {ilm} ({ilm-init}) policies to automatically manage indices 
+according to your performance, resiliency, and retention requirements. 
 For example, you could use {ilm-init} to:
 
 * Spin up a new index when an index reaches a certain size or number of documents
 * Create a new index each day, week, or month and archive previous ones
 * Delete stale indices to enforce data retention standards
-
+ 
+You can create and manage index lifecycle policies through {kib} Management or the {ilm-init} APIs.
 When you enable {ilm} for {beats} or the {ls} {es} output plugin, 
-{ilm-init} is configured automatically. 
-You can modify the default policies through {kib} Management or the {ilm-init} APIs.
+default policies are configured automatically.
+
+image:images/ilm/index-lifecycle-policies.png[]
```

Comment:
I don't think that this screenshot adds a lot of value here. There's not any context for where this screenshot exists in the UI or how to get there, so I'd suggest removing it.

Reply:
Its primary purpose is to show that there *is* a UI.

Reply:
Ah, understood! Perhaps it's worth adding a figure caption to explain the screenshot? For example:
```
.Kibana screen for managing index lifecycle policies
```